site/docs/voice/webhooks/realtimeTranscriptionRejected.mdx

```diff
@@ -0,0 +1,148 @@
+
+## Request Parameters
+TODO `s/originalTranscription/originalRealtimeTranscription`?
+
+| Property                          | Description  |
+|:----------------------------------|:-------------|
+| accountId                         | The user account associated with the call |
+| answerTime                        | Time the call was answered, in ISO 8601 format |
+| applicationId                     | The id of the application associated with the call |
+| callId                            | The call id associated with the event |
+| callUrl                           | The URL of the call associated with the event |
+| cause                             | The reason the real-time transcription was rejected, can be one of `connection-error`, `duplicate`, `error` or `track-limit` |
+| direction                         | The direction of the call. Either `inbound` or `outbound`. The direction of a call never changes. |
+| enqueuedTime                      | (optional) If [call queueing](/apis/voice#operation/createCall/) is enabled and this is an outbound call, time the call was queued, in ISO 8601 format. |
+| errorMessage                      | Detailed information about the `cause` |
+| eventTime                         | The approximate UTC date and time when the event was generated by the Bandwidth server, in ISO 8601 format. This may not be exactly the time of event execution. |
+| eventType                         | The event type, value is `realtimeTranscriptionStarted` |
+| from                              | The provided identifier of the caller: can be a phone number in E.164 format (e.g. +15555555555) or one of `Private`, `Restricted`, `Unavailable`, or `Anonymous`. |
+| realtimeTranscription             | Details about the rejected real-time transcription |
+| realtimeTranscription.name        | The name of the rejected real-time transcription |
+| realtimeTranscription.startTime   | The approximate UTC date and time the rejected real-time transcription was started |
+| realtimeTranscription.id          | The unique id of the rejected real-time transcription |
+| realtimeTranscription.tracks      | The segments of the call that were going to be sent in the rejected real-time transcription, values will be one or both of `inbound` and `outbound` |
+| realtimeTranscription.destination | (optional) The destination URL to which the rejected real-time transcription was going to be sent |
+| realtimeTranscription.stabilized  | (optional) Whether transcription update events will be sent to the specified `destination` only after they have become stable.   |
+| originalTranscription             | (optional) Details about the original real-time transcription if this rejection is due to requesting a duplicate name |
+| originalTranscription.name        | The name of the original real-time transcription |
```

Comment:
shouldn't these all also be optional?

Reply:
I used `(optional)` in the subfield to indicate whether they were optional within their parent structure. They're all optional in terms of the parent being optional.